src/events/event_handler.ts

```diff
@@ -127,6 +127,46 @@ export class eventHandler {
 		this.handleChanges(filePath, currentText);
 	}
 
+	handleTabChangeEvent = (tab_data: object) => {
+		/*
+		The flow for handler will be as follows:
+			- For the current open repoPath, get the repo_id and from config File
+			- Get Current Open Tabs TabGroups.ALL
+			- For each tab:
+				* Get the Relative Path of each file using repoPath
+				* Get the fileID using relative_path found in above step
+		*/
+
+		// For the current open repoPath, get the repo_id and from config File
+		const configJSON = readYML(this.settings.CONFIG_PATH);
+		const repo_id = configJSON.repos[this.repoPath].id;
+		console.log(`Repo ID: ${repo_id}`)
+		// Discard event if no repo_id found
+		if (!repo_id) return;
+
+		// Get list of current tabs
+		const open_tabs = vscode.window.tabGroups.all;
+		// Loop through tab groups
+		for (const tab_group of open_tabs) {
+			for (let tab of tab_group.tabs) {
+				console.log(`Displaying tabs: `, tab);
+				// Get path of tab
+				// @ts-ignore
+				let tab_path = (tab.input.uri.path).split('/');
```

Comment:
We need to make sure this works for all systems, Windows, Mac, Linux

Reply:
Good point. I have seen the following method: 

![image](https://github.com/codesyncapp/codesync-vs/assets/15974895/3b8ef620-eccb-4f86-9586-aafd408e477b)

But seems like this handles for Windows path only. Is there another method that we use to handle paths?